charts/seed-monitoring/charts/core/charts/prometheus/rules/node-exporter.rules.yaml

```
@@ -65,6 +65,19 @@ groups:
       description: Root filesystem device on instance {{ $labels.instance }} is almost full.
       summary: Node's root filesystem is almost full
 
+  # alert if the /run filesystem is almost out of inodes
+  - alert: NodeFilesystemAlmostOutOfFiles
+    expr: node_filesystem_files_free{mountpoint="/run"} / node_filesystem_files{mountpoint="/run"} * 100 < 5 and node_filesystem_readonly{mountpoint="/run"} == 0
+    for: 1h
+    labels:
+      service: node-exporter
+      severity: warning
+      type: shoot
+      visibility: owner
```

```suggestion
      visibility: all
```

If we set the `visibility` to all, both owners (see [config](https://github.com/gardener/gardener/blob/dd9c0c1c107f0de8f82a687b3e1267546f7985e3/charts/seed-monitoring/charts/alertmanager/templates/_config.tpl#L28-L30)) and Gardener [operators](https://github.com/gardener/gardener/blob/dd9c0c1c107f0de8f82a687b3e1267546f7985e3/charts/seed-bootstrap/templates/alertmanager/_config.tpl#L31-L33) will be notified, via email (assuming the alert manager is configured for the shoot owners).